### PR TITLE
Moved slackNameHandlerService from KeeperSlackCommandController to DefaultKeeperService

### PR DESCRIPTION
**Pull request by Solykk**
Connect #35 
changed KeeperServiceInterface
changed KeeperRepositoryInterface
added @EqualsAndHashCode to KeeperRequest
corrected tests after refactoring

### CODE REVIEW

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/service/impl/DefaultKeeperService.java [hunk 1]
```diff
@@ -4,50 +4,135 @@
...
         logger.info("Dismissed Keeper: [{}]", Arrays.toString(ids));
-        return ids;
+
+        String result = "ERROR. Something went wrong. Keeper was not dismissed :(";
+
+        if (ids.length > 0) {
+            result = String.format("Keeper: %s in direction: %s dismissed" ,
+                    slackParsedCommand.getFirstUserFromText().getSlack(), keeperRequest.getDirection());
+        }
+        return result;
```
Comment (NikolayNN) [inline src/main/java/ua/com/juja/microservices/keepers/slackbot/service/impl/DefaultKeeperService.java]: I think better:
if (ids.length > 0) {
            return String.format("Keeper: %s in direction: %s dismissed" ,
                    slackParsedCommand.getFirstUserFromText().getSlack(), keeperRequest.getDirection());
}else{
            return "ERROR. Something went wrong. Keeper was not dismissed :(";
}
and below

File: src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java [hunk 2]
```diff
@@ -43,22 +34,6 @@
...
-    public void setup() {
-        userFrom = new UserDTO("AAA000", "@from-user");
-        user1 = new UserDTO("AAA111", "@slack1");
-        user2 = new UserDTO("AAA222", "@slack2");
-        user3 = new UserDTO("AAA333", "@slack3");
-    }
-
     @Test
     public void onReceiveSlashCommandKeeperAddIncorrectTokenShouldReturnSorryRichMessage() throws Exception {
         final String KEEPER_ADD_COMMAND_TEXT = "@slack_name teems";
```
Comment (NikolayNN) [inline src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java]: correct - teams

File: src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java [hunk 3]
```diff
@@ -195,14 +136,11 @@ public void getKeeperDirectionsReturnEmptyRichMessage() throws Exception {
...
-        Map<String, UserDTO> users = new HashMap<>();
-        users.put(userFrom.getSlack(), userFrom);
-        users.put(user1.getSlack(), user1);
-        SlackParsedCommand slackParsedCommand = new SlackParsedCommand(userFrom.getSlack(), GET_DIRECTIONS_COMMAND_TEXT, users);
+
         // when
-        when(slackNameHandlerService.createSlackParsedCommand(userFrom.getSlack(), GET_DIRECTIONS_COMMAND_TEXT))
-                .thenReturn(slackParsedCommand);
-        when(keeperService.getKeeperDirections(any(KeeperRequest.class))).thenReturn(Arrays.asList("direction1", "direction2"));
+        when(keeperService.getKeeperDirections(any(String.class), any(String.class)))
```
Comment (NikolayNN) [inline src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java]: it is so abstact test. you should check the Strings.

**Review by NikolayNN — APPROVED** [review]
and please to do code format in the classes. You can do it automatically, use this hot key CTRL+ALT+L

**Review by NikolayNN — CHANGES_REQUESTED** [review]
and please to do code format in the classes. You can do it automatically, use this hot key CTRL+ALT+L

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/service/impl/DefaultKeeperService.java (hunk 1 above)
Comment (Solykk) [reply src/main/java/ua/com/juja/microservices/keepers/slackbot/service/impl/DefaultKeeperService.java]: fixed

File: src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java (hunk 2 above)
Comment (Solykk) [reply src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java]: fixed

File: src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java (hunk 3 above)
Comment (Solykk) [reply src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java]: fixed

**Review by NikolayNN — APPROVED** [review]

**Pull request merged**